decidim-core/lib/devise/models/decidim_validatable.rb

```diff
@@ -34,6 +34,8 @@ def self.included(base)
           validates_presence_of :password, if: :password_required?
           validates_confirmation_of :password, if: :password_required?
           validates_length_of :password, within: password_length, allow_blank: true
```

Comment:
I think you can remove the password length validation as it's already part of the password validator:
https://github.com/decidim/decidim/blob/3bfe7e17a583f2c962017c158107aa136a56dda1/decidim-core/app/validators/password_validator.rb#L10

Reply:
Ok, I agree, I've removed it and also commented it in the devise initializer